src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -944,6 +944,7 @@ def load_arguments(self, _):
             c.ignore('aux_subscriptions')
             c.argument('edge_zone', edge_zone_type)
             c.argument('accept_term', action='store_true', help="Accept the license agreement and privacy statement.")
+            c.argument('disable_integrity_monitoring', action='store_true', min_api='2020-12-01', help='Disable the default behavior (installing guest attestation extension and turning on MSI) when creating VM/VMSS compliant with Trusted Launch.')
```

Comment:
Can we change the text to "Disables the default behavior of installing the Guest Attestation Extension and System Assigned Identity for Trusted Launch enabled VMs and VMSS."

Reply:
```suggestion
            c.argument('disable_integrity_monitoring', action='store_true', min_api='2020-12-01', help='Disable the default behavior of installing guest attestation extension and enabling System Assigned Identity for Trusted Launch enabled VMs and VMSS.')
```

Reply:
Yes, it has been applied